### ISSUE TITLE
segmentation fault on version2 (ubuntu 14.04 64 bit)

### ISSUE THREAD

**Issue by kobigurk** [libbitcoin/libbitcoin-server#145]
Happened to me a few times that bitcoin-server stopped running, and the terminal showed a segmentation fault, with the following messages before it (I *think* it was always about get_blocks):
```
11:17:57.959586 INFO [poller] Block #391104 0000000000000000063067ef2e7513ea864d297c76f15e4af4caeb46c9af5618
11:18:18.855914 INFO [responder] Failure handling a get_blocks request: feature not yet supported.
11:18:18.856656 INFO [responder] Failure handling a get_blocks request: feature not yet supported.
11:18:18.856783 INFO [responder] Failure handling a get_blocks request: feature not yet supported.
11:18:18.857644 INFO [responder] Failure handling a get_blocks request: feature not yet supported.
11:18:18.857739 INFO [responder] Failure handling a get_blocks request: feature not yet supported.
11:18:18.858606 INFO [responder] Failure handling a get_blocks request: feature not yet supported.
11:18:18.858772 INFO [responder] Failure handling a get_blocks request: feature not yet supported.
11:18:18.870634 INFO [responder] Failure handling a get_blocks request: feature not yet supported.
Segmentation fault (core dumped)
```

**Comment by evoskuil** [comment]
OK, I've disabled the get_blocks query, which should have been disabled previously. This is the likely cause, since it was experimental, fairly complex and untested.

**Comment by kobigurk** [comment]
I'm running it since your fix, will reopen if I encounter it again.

**Comment by evoskuil** [comment]
The problem was reproduced on all platforms and has been resolved in v2.3.0.
